src/EnergyPlus/StandardRatings.cc

```diff
@@ -519,14 +520,11 @@ namespace StandardRatings {
         //           551/591 conditons[C]
         // par(2)  = Evaporator outlet temperature [C]
         // par(3)  = Water specific heat [J/(kg*C)]
-        // par(4)  = Part load ratio
-        // par(5)  = Evaporator mass flow rate [kg/s]
-        // par(6)  = Index for the total cooling capacity modifier curve
-        // par(7)  = Index for the energy input ratio modifier curve
-        // par(8)  = Index for the EIR vs part-load ratio curve
-        // par(9)  = Reference capacity of chiller [W]
-        // par(10) = Reference coefficient of performance [W/W]
-        // par(11) = Open chiller motor efficiency [fraction, 0 to 1]
+        // par(4)  = Percent load per AHRI: The ratio of the part-load net capacity to the full-load rated net capacity at the full-load rating
+        // conditions par(5)  = Condenser mass flow rate [kg/s] par(6)  = Index for the total cooling capacity modifier curve par(7)  = Index for the
+        // energy input ratio modifier curve par(8)  = Index for the EIR vs part-load ratio curve par(9)  = Reference capacity of chiller [W] par(10)
+        // = Reference coefficient of performance [W/W] par(11) = Open chiller motor efficiency [fraction, 0 to 1] par(12) = CAP-f-T at
+        // reference/rated temperatures
```

Comment:
Old formatting "looked" better, but this is a result of applying clang.

Reply:
Actually the par(4) text exceeds the max line length. If the word "the" was removed in 3 places this would revert back (with a little help). No need to do anything about this now.